data/json/recipes/food/brewing.json

```diff
@@ -189,7 +189,7 @@
         "components": [
             [ [ "water", 3 ], [ "water_clean", 3 ] ],
             [ [ "raw_dandelion", 10 ] ],
-            [ [ "sugar", 10 ], [ "honeycomb", 1 ], [ "honey_bottled", 2 ], [ "honey_glassed", 2 ] ],
+            [ [ "sugar", 10 ], [ "honeycomb", 1 ], [ "honey_bottled", 2 ], [ "honey_glassed", 2 ], [ "syrup", 1 ] ],
```

Comment:
1 syrup but 2 honey. Syrup isn't significantly sweeter than honey, so it shouldn't get lower cost.